### PR TITLE
Adds support to multiple ranges in ctl:ruleRemoveTargetById (#2110)

### PR DESCRIPTION
**Pull request by j0k2r**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,6 +1,8 @@
 v3.0.4 - YYYY-MMM-DD (to be released)
 -------------------------------------
 
+ - Adds support to multiple ranges in ctl:ruleRemoveTargetById
+   [Issue #2110 - @theMiddleBlue, @zimmerle]
```
Comment (zimmerle) [inline CHANGES]: @j0k2r why you did put my name on it and where is yours? :)

Comment (j0k2r) [reply CHANGES]: Oh, sorry I don't know the ModSecurity CHANGES rules, I took the the issue "participants"

**Comment by dune73** [comment]
Is this PR covering the use of multiple comma-separated IDs too? (-> `ctl:ruleRemoveTargetById=1000,1999`)

If it does, then adding a test case for that use would be helpful.

**Comment by j0k2r** [comment]
No, it only parse the Id range

**Comment by j0k2r** [comment]
@dune73, After doing some tests, it seems that neither `ctl:ruleRemoveById` nor `ctl:ruleRemoveTargetById` supports setting multiple IDs.

I'm not a Lexer expert but apparently with the https://github.com/SpiderLabs/ModSecurity/blob/cbd15ec1382f7e2c3e19a93ea9ca3e8b4ced44f6/src/parser/seclang-scanner.ll#L447 **ruleRemoveBy** does not support (,)

Don't know if I'm right.

**Comment by dune73** [comment]
Thank you for checking. Much appreciated. So many subtle shortcomings with v3...

**Comment by zimmerle** [comment]
Thank you @j0k2r. Added this as part of #2430. Merged on 3.1-experimental

**Pull request closed without merging**